spherical.c

```diff
@@ -111,7 +111,17 @@ int spherical(int index_operation, double yield_strength,
                 printf("Method yet to be implemented\n");
                 break;
             case 3:
-                printf("Method yet to be implemented\n");
+		if(fos_uts*pressure < 0.665*ultimate_strength*weld_efficiency)
+		{
+		t_bpvc= fos_uts*pressure*internal_diameter/\
```

Comment:
1. Add the UG section numbers for the formulas used.
2. Why max. allowable stress is UTS? why not YS? 
3. One more condition is given in UG 27 for spherical shells - t < 0.356R. Does it need to be checked after evaluating the thickness?

Reply:
corrected